### PR TITLE
OHRM5X-1347: Develop corporate branding - theme preview API

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

File: installer/Migration/V5_1_0/lang-string/general.yaml [hunk 1]
```diff
@@ -145,7 +145,7 @@ langStrings:
     unitId: preview
   -
     value: 'Accepts jpg, .png, .gif up to {fileSize}. Recommended dimensions: {width}px X {height}px'
-    unitId: accept_jpg_png_gif_upto_recomended_dimentions
+    unitId: accept_jpg_png_gif_upto_recommended_dimensions
```
Comment (SharukaNP) [inline installer/Migration/V5_1_0/lang-string/general.yaml:148]: Hope that the correction of this unitId is applied in places where it was previously used as well.

File: src/client/src/orangehrmAdminPlugin/pages/corporateBranding/CorporateBranding.vue [hunk 2]
```diff
@@ -133,7 +133,7 @@
                 :file="loginBanner.oldAttachment"
                 :rules="rules.loginBanner"
                 :hint="
-                  $t('general.accept_jpg_png_gif_upto_recomended_dimentions', {
+                  $t('general.accept_jpg_png_gif_upto_recommended_dimensions', {
```
Comment (SharukaNP) [inline src/client/src/orangehrmAdminPlugin/pages/corporateBranding/CorporateBranding.vue:136]: good

File: src/plugins/orangehrmCorporateBrandingPlugin/test/Api/ThemeAPITest.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    /**
+     * @return array
+     */
+    public function dataProviderForTestGetOne(): array
+    {
+        return $this->getTestCases('ThemeAPITestCases.yaml', 'GetOne');
+    }
+
+    public static function keepOnlyDefaultThemePreHook(): void
```
Comment (SharukaNP) [inline src/plugins/orangehrmCorporateBrandingPlugin/test/Api/ThemeAPITest.php:55]: Is this function being used?

Comment (RajithaKumara) [reply src/plugins/orangehrmCorporateBrandingPlugin/test/Api/ThemeAPITest.php:55]: yes,
https://github.com/orangehrm/orangehrm/blob/a1c5c5e7bf9ef6fa0a9b0bc5edd4245a07174e27/src/plugins/orangehrmCorporateBrandingPlugin/test/fixtures/testcases/ThemeAPITestCases.yaml#L32

**Comment by RajithaKumara** [comment]
Related: #1323 #1330

**Pull request merged**